### PR TITLE
Comments on articles

### PR DESCRIPTION
**Pull request by AntonellaMorittu**
#### Description

User can add comment under articles.

Changes proposed in this pull request:

* Add comment section under article
* Display comment under article
* Add associations between article and comment
* Add database table for comments

#### What I have learned working on this feature: 
* How to add create and change db table
* How to add associations 
* How to properly use binding.pry

schermata 2018-03-19 alle 10 44 19

### CODE REVIEW

File: app/models/article.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
 class Article < ApplicationRecord
   validates :title, :content, presence: true
```
Comment (faraznaeem) [inline app/models/article.rb:2]: You are validating the presence of the content, However, I can't see any sad path or test for these

**Review by faraznaeem — CHANGES_REQUESTED** [review]

File: app/models/article.rb (hunk 1 above)
Comment (NellaMori) [reply app/models/article.rb:2]: You wanted to comment the comment.rb model instead?We have just created and tested it :)

File: features/user_can_comment_on_article.feature [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  Scenario: Successfully creates a comment on an article
+    Given I am on the "A Whole New World" page
+    When I fill in "Email" with "antonella@email.com"
+    And I fill in "Comment" with "This article is fun"
+    And I click "Submit comment" button
+    Then I should see "antonella@email.com"
+    And I should see "This article is fun"
+
+  Scenario: User edits the article and leave content empty
```
Comment (davidboll) [inline features/user_can_comment_on_article.feature]: Is the user editing the comment or the article?

**Review by faraznaeem — APPROVED** [review]

File: features/user_can_comment_on_article.feature [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    Given I am on the "A Whole New World" page
+    When I fill in "Email" with "antonella@email.com"
+    And I fill in "Comment" with ""
+    And I click "Submit comment" button
+    Then I should be on "A Whole New World" page
+    And I should see "Field cannot be blank."
+
+  Scenario: User edits the article and leave email empty
```
Comment (davidboll) [inline features/user_can_comment_on_article.feature]: Same as above, is the user editing the article or the comment?

**Review by davidboll — CHANGES_REQUESTED** [review]

File: features/user_can_comment_on_article.feature (hunk 2 above)
Comment (NellaMori) [reply features/user_can_comment_on_article.feature]: really good catch @davidboll !thanks, fixed!

**Review by oliverochman — APPROVED** [review]
LGTM!!!

**Review by davidboll — APPROVED** [review]

**Pull request merged**